### PR TITLE
Add template disks

### LINKED ISSUE #32: Feature Request: Multiple Disk Support
Would be useful to be able to supply a variable containing a lists of [disks](https://github.com/oVirt/ovirt-ansible-image-template/blob/master/tasks/qcow2_image.yml#L118) which should be attached to vm before template is made so that multiple disks could be part of the template instead of a single disk.

The presence of the disks would be the user's responsibility, but attaching disks from within this role via role level variable might be useful for those making templates which references many disks.

### PR DESCRIPTION
**Pull request by mnecas**
Fixes #32 
Added variable template_disks. Its for adding multiple disks to template.
@machacekondra @mwperina

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -65,6 +66,20 @@ Role Variables
 | glance_image            | UNDEF (mandatory if qcow_url is not used)               | This parameter specifies the name of disk in glance provider to be imported as template. |
 | template_prerequisites_tasks | UNDEF | Works only with qcow image. Specify a path to Ansible tasks file, which should be executed on virtual machine before creating a template from it. Note that qcow image must contain guest agent which reports IP address. |
 
+The `template_disks` List of dictionaries can contain following attributes:
+
+| Name               | Default value  |                                              |
+|--------------------|----------------|----------------------------------------------|
+| size               | UNDEF          | The size of the additional disk. |
+| name               | UNDEF          | The name of the additional disk.  |
```
Comment (mnecas) [inline README.md]: add required

File: tasks/qcow2_image.yml [hunk 2]
```diff
@@ -133,6 +133,19 @@
               interface: "{{ template_disk_interface }}"
           nics: "{{ template_nics }}"
 
+      - name: Manage disks
+        ovirt_disk:
+          auth: "{{ ovirt_auth }}"
+          name: "{{ item.name }}"
+          vm_name: "{{ vm_name }}"
+          size: "{{ item.size | default(omit) }}"
```
Comment (mnecas) [inline tasks/qcow2_image.yml:144]: item.size | mandatory

File: tasks/qcow2_image.yml [hunk 3]
```diff
@@ -133,6 +133,19 @@
               interface: "{{ template_disk_interface }}"
           nics: "{{ template_nics }}"
 
+      - name: Manage disks
+        ovirt_disk:
+          auth: "{{ ovirt_auth }}"
+          name: "{{ item.name }}"
+          vm_name: "{{ vm_name }}"
```
Comment (mnecas) [inline tasks/qcow2_image.yml]: item.size | mandatory

File: tasks/qcow2_image.yml [hunk 4]
```diff
@@ -133,6 +133,19 @@
               interface: "{{ template_disk_interface }}"
           nics: "{{ template_nics }}"
 
+      - name: Manage disks
+        ovirt_disk:
+          auth: "{{ ovirt_auth }}"
+          name: "{{ item.name }}"
```
Comment (mnecas) [inline tasks/qcow2_image.yml]: item.name | madatory

File: README.md [hunk 5]
```diff
@@ -65,6 +66,20 @@ Role Variables
 | glance_image            | UNDEF (mandatory if qcow_url is not used)               | This parameter specifies the name of disk in glance provider to be imported as template. |
 | template_prerequisites_tasks | UNDEF | Works only with qcow image. Specify a path to Ansible tasks file, which should be executed on virtual machine before creating a template from it. Note that qcow image must contain guest agent which reports IP address. |
 
+The `template_disks` List of dictionaries can contain following attributes:
+
+| Name               | Default value  |                                              |
+|--------------------|----------------|----------------------------------------------|
+| size               | UNDEF (Required)          | The size of the additional disk. |
+| name               | UNDEF (Required)          | The name of the additional disk.  |
+the disk will be looked up by id and will update name of the disk if it differs from the name passed in name parameter. |
```
Comment (mwperina) [inline README.md]: `The disk will be looked up by id and its name will be updated if it differs from the name passed in name parameter`

**Review by mwperina — CHANGES_REQUESTED** [review]
Looks good, just small rephrase in documentation

**Review by mwperina — APPROVED** [review]
+1

**Pull request merged**